pkgs/applications/editors/glow/default.nix

```diff
@@ -3,19 +3,27 @@
 , fetchFromGitHub
 , installShellFiles
 , stdenv
+, fetchpatch
 }:
 buildGoModule rec {
   pname = "glow";
-  version = "1.5.0";
+  version = "1.5.1";
 
   src = fetchFromGitHub {
     owner = "charmbracelet";
     repo = "glow";
     rev = "v${version}";
-    sha256 = "sha256-CI0S9XJtJQClpQvI6iSb5rcHafEUwr2V6+Fq560lRfM=";
+    sha256 = "sha256-12UziCf3BO1z+W02slNCCvXhIkvZuVgXk++BdHG3gDI=";
   };
 
-  vendorHash = "sha256-2QrHBbhJ04r/vPK2m8J2KZSFrREDCc18tlKd7evghBc=";
+  vendorHash = "sha256-VLuhkpibcm7t0lDMR+tZyV+tMZeIqe58UuRiU0Drjgw=";
+
+  # Remove whenever https://github.com/charmbracelet/glow/pull/496 is merged and a release with it is available
+  patches = [(fetchpatch {
+    url = "https://github.com/charmbracelet/glow/pull/496.patch";
+    name = "go-1-17-patch";
+    hash = "sha256-J6zjOdWz2P7/AGbV5cAQiC6n4ndWWAeilcBuDJoTZcQ=";
+  })];
```

Comment:
Can we use this commit instead?
https://github.com/charmbracelet/glow/commit/f0734709f0be19a34e648caaf63340938a50caa2

Also, the diff of the pull request may change, so please get the patch from a commit.

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/257931/commits/abbdc72d08ff5773aa2791bfb22ebdd1b88cff6f